scripts/check-asciidoctor-build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+
+# ensure asciidoctor is installed
+if ! command -v asciidoctor ;
+then
+    echo "Asciidoctor is not installed. Please install it and try again."
+    exit 1
```

Comment:
This isn't strictly necessary, but different exit codes for different failure states might be a good idea. Right now, the only failure code exited with is `1` whether the script exits due to `asciidoctor` being unavailable or due to actual build errors.

Reply:
Updated with exit code 127